From 4554fb5a1b18017e35a7ba63517a409ea438876d Mon Sep 17 00:00:00 2001
From: Michael Abbott <michael.abbott@diamond.ac.uk>
Date: Wed, 6 May 2009 12:47:22 +0100
Subject: [PATCH] Export GPIO control symbols from mfp-pxa2xx.c

This allows modules to configure their own GPIO settings if this hasn't
been done at the board level.

Signed-off-by: Michael Abbott <michael.abbott@diamond.ac.uk>
---
 arch/arm/mach-pxa/mfp-pxa2xx.c |    5 +++++
 1 files changed, 5 insertions(+), 0 deletions(-)

diff --git a/arch/arm/mach-pxa/mfp-pxa2xx.c b/arch/arm/mach-pxa/mfp-pxa2xx.c
index 33626de..3f2007a 100644
--- a/arch/arm/mach-pxa/mfp-pxa2xx.c
+++ b/arch/arm/mach-pxa/mfp-pxa2xx.c
@@ -384,3 +384,8 @@ static int __init pxa2xx_mfp_init(void)
 	return sysdev_class_register(&pxa2xx_mfp_sysclass);
 }
 postcore_initcall(pxa2xx_mfp_init);
+
+
+EXPORT_SYMBOL(pxa2xx_mfp_config);
+EXPORT_SYMBOL(pxa2xx_mfp_set_lpm);
+EXPORT_SYMBOL(gpio_set_wake);
-- 
1.6.1.3

